### PR TITLE
New app manager

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

**Comment by yakutovicha** [comment]
also solves #8

**Comment by yakutovicha** [comment]
@ltalirz, the local repository problem should be now fixed. please try again.

File: aiidalab_logo_v4.svg [hunk 1]
```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
[the remark this answers is not in the document]
Comment (yakutovicha) [reply aiidalab_logo_v4.svg]: Is this logo rotating?

[the remark this answers is not in the document]
Comment (yakutovicha) [reply aiidalab_logo_v4.svg]: anyways, if I use HTML for that - it won't work ( I can't point to the file which is located in `/usr/local/share/jupyter/nbextensions/appmode/gears.svg`)

**Comment by yakutovicha** [comment]
@ltalirz, I implemented all what I planned for today. Please have a look and let me know if there are issues.

**Comment by ltalirz** [comment]
@yakutovicha I'm using the "update" functionality to update the home app and when I click on the button, I get "success" but then it remains orange.
I can click on the button again and again and the same happens.

**Comment by yakutovicha** [comment]
Yes, because [apps_meta.json](https://github.com/aiidalab/aiidalab-registry/blob/gh-pages/apps_meta.json) is not automatically updated with the new information from the repository

**Comment by yakutovicha** [comment]
> I'm using the "update" functionality to update the home app and when I click on the button, I get "success" but then it remains orange.

Now should be fixed.

**Pull request merged**